zookeeper/source/zk_nn_client.cc

```diff
@@ -1333,11 +1337,27 @@ bool ZkNnClient::add_block_group(const std::string &fileName,
     }
 
     // Log them for debugging purposes.
+    std::vector<std::shared_ptr<ZooOp>> ops;
     LOG(INFO) << "Generated block group id " << block_group_id << "\n";
-    for (auto storageBlockID : storageBlockIDs)
-        LOG(INFO) << "Generated storage block id " << storageBlockID << "\n";
-
-    // TODO(nate): figure out what exact zookeepr operations must occur.
+    auto block_location_op = zk->build_create_op(
+            "/block_locations/" + std::to_string(block_group_id), ZKWrapper::EMPTY_VECTOR);
```

Comment:
As was proposed in my PR (https://github.com/comp413-2017/RDFS/pull/113), I suggest that we keep two separate path prefixes, "/block_locations/" for non EC blocks and "/block_group_locations/" for EC blocks.